docs/getting-started-kubernetes.rst

```diff
@@ -1896,6 +1896,13 @@ Once the Network Operator has been installed create a NicClusterPolicy with:
     metadata:
       name: nic-cluster-policy
     spec:
+      ofedDriver:
```

Comment:
do we really need this here ? im fine with having switchdev example without MOFED.

main use-cases for MOFED we have are for GPU Direct RDMA often this is not mixed with switched.